packages/tools/cookbook/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@kadena/cookbook",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "scripts": {
+    "build": "",
+    "test": ""
+  },
+  "dependencies": {
+    "@kadena/client": "workspace:*",
+    "@kadena/chainweb-node-client": "workspace:*"
+  },
+  "devDependencies": {
+    "@types/node": "^16.0.0",
+    "@kadena/pactjs-cli": "workspace:*",
+    "@kadena/heft-rig": "workspace:*"
+  }
```

Comment:
Can we add the `lint-staged` command? See [for reference](https://github.com/kadena-community/kadena.js/blob/feat/cookbook/packages/libs/bootstrap-lib/package.json#L35-L43)

We probably want to have the whole package json look like the one from `bootstrap-lib`

Reply:
Took what seemed like the relevant items from bootstrap-lib!